### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by jlwllmr**
- Reorganise content, consolidating material that was relevant to the contribution section
- Add some details around how partner organisations should submit contributions.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,19 +91,27 @@ It is highly recommended that you use a tool like [`nvm`](https://github.com/nvm
     $ npm run prepare
     $ npm start
 
-This command starts a local development server and opens up a browser window. Most changes are reflected live without having to restart the server.
+This command starts a local development server and opens up a browser window. Most changes are
```
Comment (bgravenorst) [inline README.md:94]: Duplicate content here. "Running this project" and "Local Development"  have the same steps.

File: README.md [hunk 2]
```diff
@@ -57,19 +91,27 @@ It is highly recommended that you use a tool like [`nvm`](https://github.com/nvm
...
+that is not known to the linter, and your build, or commit, will fail.
 
-If this happens, take a look at `project-words.txt` in the root directory of your project; if the word that the linter caught is correctly spelled, and you wish it to pass the linter's test, add it to `project-words.txt`, save, add and commit those changes, and see if it passes.
+If this happens, take a look at `project-words.txt` in the root directory of your project; if the 
+word that the linter caught is correctly spelled, and you wish it to pass the linter's test, 
+add it to `project-words.txt`, save, add and commit those changes, and see if it passes.
+
+For tidiness, please ensure you adhere to the alphabetical order established in `project-words.txt`.
 
 ### Local development
```
Comment (bgravenorst) [inline README.md:116]: Duplicate content? This appears to be the same as the "Local Development" section above.

File: README.md [hunk 3]
```diff
@@ -1,28 +1,62 @@
...
 Take a look at some [good first issues](https://github.com/ConsenSys/doc.linea/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) to get started.
 
 ### How to submit a suggestion or change
 
-The best way to suggest a change to these docs is through a process known as a **pull request**. If you're not familiar with how that works, check out [GitHub's guide here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
+The best way to suggest a change to these docs is through a process known as a **pull request**. 
+If you're not familiar with how that works, check out [GitHub's guide here](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request).
 
-If that process is too involved for you, you can always open a thread on the [Community forum](https://community.linea.build/), or a ticket on the [Support page](https://support.linea.build/hc/en-us).
+If you do intend to submit a pull request, please open an issue first, and link to it in your pull
```
Comment (bgravenorst) [inline README.md:26]: Perhaps mention how to link it to the PR?

File: README.md [hunk 4]
```diff
@@ -1,28 +1,62 @@
...
+details in an issue first will make it much easier for our docs team to process your contributions.
 
-If you **are** familiar with making a pull request, we **highly recommend that you run a version of these docs locally, and preview your changes locally, before submitting them**. In fact, it's part of the PR process.
+If that process is too involved for you, you can always open a thread on the [Community forum](https://community.linea.build/), 
+or a ticket on the [Support page](https://support.linea.build/hc/en-us).
+
+If you are familiar with making a pull request, we **highly recommend that you run a version of these docs locally, and preview your changes locally, before submitting them**. 
+In fact, it's part of the PR process.
+
+### Contribute to community tutorials
```
Comment (bgravenorst) [inline README.md:36]: Should they create issues first?

**Review by bgravenorst — COMMENTED** [review]
Added some feedback.

File: README.md (hunk 4 above)
Comment (jlwllmr) [reply README.md:36]: A good idea!

**Pull request merged**